code/script.js

```diff
@@ -30,18 +74,130 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+// Interaction 7, win-win.
+// As bot, hand over food item to user and wish them a happy meal (what's the English phrase?)
+const done = () => {
+  showMessage('Here you go, buon appetito! Welcome back anytime you want!', 'bot')
+}
+
+// Interaction 6, final confirmation.
+// As bot, tell user they've chosen how to pay and that it's time to confirm wether to go through with order (if so: define, store & pass on decision), or to start over (go back to the beginning).
+const exitStrategy = (trade) => {
+  showMessage(
+    `Okay, once payment by ${trade} is done, we'll have your meal ready in no time. You ready to go through?`,
+    'bot')
+  inputWrapper.innerHTML = exitHTML
+  
+  const yes = document.getElementById('yes')
+  yes.addEventListener('click', () => {
+    showMessage(`Yup, GET IN MY BELLY!`, 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => done(yes), 1000)
+  })
+  document
+  .getElementById('no')
+  .addEventListener('click', () => {
+    showMessage(`Let's start over, I need to change my order...`, 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => window.location.reload(), 1000)
+  })
+}
+
+// Interaction 5, food decision has been made.
+// As bot, reinforce choice. Ask user for payment, and present options the user may choose from. Pass the choice of payment on.
+const payment = (dish) => {
+  showMessage(
+    `Pristine choice! It'll be 100 SEK for your ${dish}. How would you like to pay?`,
+    'bot')
+  inputWrapper.innerHTML = paymentHTML
+  document
+  .getElementById('card')
+  .addEventListener('click', () => {
+    showMessage(`Plastic (not so) fantastic.`, 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => exitStrategy('card'), 1000)
+  })
+  document
+  .getElementById('cash')
+  .addEventListener('click', () => {
+    showMessage(`I have a mint bill here for you... 🤑`, 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => exitStrategy('cash'), 1000)
+  })
+  document
+  .getElementById('dishes')
+  .addEventListener('click', () => {
+    showMessage(`See, I can't... But I'll do the dishes! Please?`, 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => exitStrategy('doing the dishes'), 1000)
+  })
 }
 
-// Set up your eventlisteners here
+// Interaction 4, choose what to eat.
+// As bot, confirm choice. Depending on choice, show specific menu the user may select as a drop-down. Pass the value of this choice on.
+const mainChoice = (specificChoice) => {
+  if (specificChoice === 'pizza') {
+  showMessage(`Excellent, pizza's always a winner! Make your choice below.`, 'bot')
+  inputWrapper.innerHTML = selectPizza
+  } else if (specificChoice === 'pasta') {
```

Comment:
Your menu variables keep the `mainChoice` function really clean ⭐ You can indent the code in the conditionals as well, like this:
```
if (specificChoice === 'pizza') {
  showMessage(`Excellent, pizza's always a winner! Make your choice below.`, 'bot')
  inputWrapper.innerHTML = selectPizza
}
```